README.md

```diff
@@ -28,7 +28,7 @@ Clone demo application repository.
 
 #### Step 3
 
-Find ``MainActivity.kt`` and replace ``<vault_id>`` with your organization
+Find ``MainActivity.kt`` and place your ``<vault_id>`` with your organization
```

Comment:
why did your change that?

Reply:
some time ago we change ``<vault_id>`` declaration on real VaultID but I don't remember when and why.
At the current moment we init vault in the next way:

```
private const val VAULT_ID = "tntstwggghg"
//..........
vgsForm = VGSCollect(this, VAULT_ID, ENVIRONMENT)
```

So, I think ``replace `` could confuse people.